src/main/java/frc/robot/Constants.java

```diff
@@ -30,6 +30,16 @@ public class PathPlanner {
         
     }
 
+    public class Shooter {
+        public static final int INDEXER_MOTOR_ONE = 17;
+        public static final int INDEXER_MOTOR_TWO = 18;
+        public static final int SHOOTER_MOTOR_ONE = 19;
+        public static final int SHOOTER_MOTOR_TWO = 20;
+        public static final int LASER_CAN = 21; // PLACEHOLDER
```

Comment:
can we come up with a better name for this? it would be like calling any limit switch `switch`. Try to name it based on what it will do